data/avro-schema/src/main/resources/avro/net/corda/data/membership/p2p/VerificationRequest.avsc

```diff
@@ -0,0 +1,39 @@
+{
+  "type": "record",
+  "name": "VerificationRequest",
+  "namespace": "net.corda.data.membership.p2p",
+  "doc": "Request from MGM to verify our identity's membership data.",
+  "fields": [
+    {
+      "name": "destination",
+      "doc": "Holding identity of the member who needs to be verified.",
+      "type": "net.corda.data.identity.HoldingIdentity"
+    },
+    {
+      "name": "source",
+      "doc": "Holding identity of the MGM.",
+      "type": "net.corda.data.identity.HoldingIdentity"
+    },
+    {
+      "name": "requestId",
+      "doc": "ID of the request.",
+      "type": {
+        "type": "string",
+        "logicalType": "uuid"
+      }
+    },
+    {
+      "name": "requestTimestamp",
```

Comment:
I wonder is there any value for `requestId` and `requestTimestamp` in the request/response and `responseTimestamp` in the response. It's useful for the RPC message flow because we send and receive the request and response in the same process and we can compare the values. But for P2P we hand the request to the P2P layer and forget about it and later receive the response so we can't compare values between the two.